Source_Code/TM_UnitTests/TeamMentor.UnitTests/CoreLib/Schemas/Test_Schemas_Validation.cs

```diff
@@ -75,15 +75,17 @@ public void Validation_NewUser_LargeDataInFields()
                              "mappings : {0}".format(result.Value.toString()));
         }
         
-        [Test]
+        [Test,Ignore]

```

Comment:
That test was designed to test the regular expression from the annotations via tmUser.validate().
However, the regular expression is not longer used .